builder/proxmox/common/step_start_vm.go

```diff
@@ -254,6 +250,30 @@ func (s *stepStartVM) Run(ctx context.Context, state multistep.StateBag) multist
 	return multistep.ActionContinue
 }
 
+func generateAgentConfig(agent config.Trilean) *proxmox.QemuGuestAgent {
```

Comment:
Do we plan to add more to this function? It seems to me that this could be simplified to `&proxmox.QemuGuestAgent{Enable:agent.True()}` otherwise

Reply:
I did consider `proxmox.QemuGuestAgent{Enable:agent.True()}` at first but I had a look over the open issues and decided to set this up as a function so I could address #276 in another PR